index.css

```diff
@@ -120,3 +119,114 @@ h2 {
     border-radius: 50px;
     margin-top: 10px;
 }
+
+
+/***DESKTOP***/
```

Comment:
How do you feel about aligning the text above each bubble to the left? Since the larger titles are left aligned it might make it more cohesive